modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonClientCodegen.java

```diff
@@ -1087,7 +1087,7 @@ private String toExampleValueRecursive(String modelName, Schema schema, Object o
         } else if (ModelUtils.isArraySchema(schema)) {
             if (objExample instanceof Iterable) {
                 // If the example is already a list, return it directly instead of wrongly wrap it in another list
-                return fullPrefix + objExample.toString();
+                return fullPrefix + objExample.toString() + closeChars;
```

Comment:
Why are we not seeing a sample change?
Can you add a test sample which uses this code path?

Reply:
I added a test sample, and it made me realize it didn't fix it the way I expected it to be fixed. I changed the code, let me know if it makes sense.